### DISCUSSION TITLE
Getting repitative words?

### DISCUSSION THREAD

**Discussion by rizwanishaq** [openai/whisper#1974] [General]
I am testing, and getting sometime the results like "no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no" and like "chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel chanel" how can we handle this problem

**Comment by Purfview** [comment]
Most likely because of the bug mentioned there -> https://github.com/openai/whisper/pull/1903

Test that bugfix:

Install unmerged #1903 PR with Git [not sure if command is correct]:
```
python -m pip install --upgrade --no-deps --force-reinstall git+https://github.com/openai/whisper.git@refs/pull/1903/head
```

Alternative method is to install from the downloaded zip'ed repo:

`python -m pip install --upgrade --force-reinstall "whisper-patch-1.zip"`
[ downloaded from [https://github.com/Purfview/whisper/tree/patch-1](https://github.com/Purfview/whisper/tree/patch-1) ]

Or just edit "transcribe.py" yourself, it's just the two additional lines.
